packages/dds/tree/src/simple-tree/schemaTypes.ts

```diff
@@ -382,14 +383,17 @@ export type ImplicitFieldSchema = FieldSchema | ImplicitAllowedTypes;
 
 /**
  * Converts ImplicitFieldSchema to the corresponding tree node's field type.
+ * @remarks
```

Comment:
Might make sense for this to be a @typeParam, since it's all about `TSchema`.

Reply:
done.